typescript/formsApps.d.ts

```diff
@@ -310,6 +310,13 @@ type _NewFormsApp = {
   enableSamlIdentityProviderLogout?: boolean
   /** The exact forms app environment identifier the forms app is associated with */
   formsAppEnvironmentId: number
+  /**
+   * The email addresses set in `notificationEmailAddresses` will override the
+   * email addresses set at the environment level. Set this flag to `true` to
+   * include the environment level email addresses as well as the email
+   * addresses set in `notificationEmailAddresses`.
+   */
+  isExtendingEnvironmentNotificationEmailAddresses?: boolean
```

Comment:
Isn't the default behaviour of this to be true?
Seems like the comment should be worded to explain what setting to false will do.

Reply:
All existing apps will override environment config after deployment. This property means we do not have to migrate any data. The expectation is that we will set this to `true` when creating apps in the console.